app/views/runs/_split_table.slim

```diff
@@ -3,6 +3,8 @@
   .table-responsive: table.card-body.table.mb-0
     thead: tr
       th.align-right #
+      - if run.segments.map(&:icon).any?(&:attached?)
```

Comment:
🚨 n+1 alert 🚨

Reply:
This should be covered by the includes on the run class that is being passed in.  Is it still N+1'ing?

Reply:
Ah ok, I only ran the line in isolation from a Rails console. If it doesn't look like n+1 when loading the page, then disregard.

Reply:
Yeah, it's a bit misleading because I need to attach the includes to the Run class call, but that's in the controller I believe, so in the future this may become an N+1 if someone forgets it.